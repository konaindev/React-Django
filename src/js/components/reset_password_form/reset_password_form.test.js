import React from "react";
import renderer from "react-test-renderer";

import ResetPasswordForm from "./index";

describe("ResetPasswordForm", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<ResetPasswordForm></ResetPasswordForm>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
