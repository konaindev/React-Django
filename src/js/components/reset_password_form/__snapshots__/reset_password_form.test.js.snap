// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResetPasswordForm renders correctly 1`] = `
<div
  className="page-auth"
>
  <div
    className="page-auth__header"
  >
    <a
      className="page-auth__back-link"
      href="/"
    >
      ← Go to Login
    </a>
    <div
      className="page-auth__logo"
    >
      <a
        href="/dashboard"
      >
        <div
          className="remarkably-logo"
        >
           
        </div>
      </a>
    </div>
  </div>
  <div
    className="page-auth__body"
  >
    <div
      className="account-form reset-password"
    >
      <div
        className="account-form__title"
      >
        Reset my password
      </div>
      <div
        className="account-form__subtitle"
      >
        Enter your email address below and we’ll send a special link to get you back on track.
      </div>
      <div
        className="account-form__body"
      >
        <form
          method="post"
          onReset={[Function]}
          onSubmit={[Function]}
        >
          <div
            className="form-field"
          >
            <div
              className="form-field__error"
            />
            <div
              className="form-field__label"
            >
              Email:
            </div>
            <div
              className="form-field__input-container"
            >
              <input
                className="input input--highlight"
                name="email"
                onBlur={[Function]}
                onChange={[Function]}
                type="email"
              />
              <svg
                aria-hidden="true"
                className="form-field__icon form-field__icon--error"
                color="inherit"
                focusable="false"
                height="1em"
                role="presentation"
                viewBox="0 0 14 14"
                width="1em"
              >
                <rect
                  fill="white"
                  height="6"
                  width="6"
                  x="4"
                  y="4"
                />
                <path
                  clipRule="evenodd"
                  d="M0 7C0 3.13506 3.13506 0 7 0C10.8649 0 14 3.13506 14 7C14 10.8649 10.8649 14 7 14C3.13506 14 0 10.8649 0 7ZM9.42026 8.83969L8.83794 9.42027C8.73162 9.52658 8.55823 9.52658 8.45191 9.42027L6.99939 7.96799L5.54686 9.42027C5.44054 9.52658 5.26715 9.52658 5.16083 9.41864L4.57851 8.83969C4.47383 8.73174 4.47383 8.56002 4.57851 8.45208L6.03104 6.9998L4.58015 5.54915C4.47383 5.44121 4.47383 5.26785 4.58015 5.16154L5.16083 4.58096C5.26715 4.47301 5.44218 4.47301 5.5485 4.58096L6.99939 6.03161L8.45191 4.58096C8.55823 4.47301 8.73325 4.47301 8.83794 4.58096L9.42026 5.15991C9.52658 5.26785 9.52658 5.44121 9.42026 5.54915L7.96774 6.9998L9.42026 8.45208C9.52494 8.56002 9.52494 8.73174 9.42026 8.83969Z"
                  fill="#FF540D"
                  fillRule="evenodd"
                />
              </svg>
              <svg
                aria-hidden="true"
                className="form-field__icon form-field__icon--ok"
                color="inherit"
                focusable="false"
                height="1em"
                role="presentation"
                viewBox="0 0 14 14"
                width="1em"
              >
                <rect
                  fill="white"
                  height="7"
                  width="6"
                  x="4"
                  y="3"
                />
                <path
                  clipRule="evenodd"
                  d="M0 7C0 3.13506 3.13506 0 7 0C10.8649 0 14 3.13506 14 7C14 10.8649 10.8649 14 7 14C3.13506 14 0 10.8649 0 7ZM9.94652 5.04456L6.87461 9.57471C6.78379 9.70919 6.63884 9.8 6.49564 9.8C6.35243 9.8 6.19526 9.72141 6.09397 9.62187C5.99268 9.52232 4.29169 7.81784 4.29169 7.81784C4.16944 7.69559 4.16944 7.49476 4.29169 7.37251L4.73702 6.92718C4.85926 6.80493 5.0601 6.80493 5.1806 6.92718L6.35418 8.09901L8.90392 4.33727C9.00172 4.19407 9.19906 4.15739 9.34226 4.25345L9.86444 4.60796C10.0059 4.70402 10.0443 4.90136 9.94652 5.04456Z"
                  fill="currentColor"
                  fillRule="evenodd"
                />
              </svg>
            </div>
          </div>
          <button
            className="button button--block button--uppercase button--primary reset-password__button"
            onClick={[Function]}
            type="submit"
          >
            Set Password
          </button>
        </form>
      </div>
    </div>
  </div>
  <div
    className="page-auth__footer"
  >
    © 2019 Remarkably. All Rights Reserved
  </div>
</div>
`;
